style/base.css

```diff
@@ -148,3 +148,64 @@
   background: var(--jp-layout-color2);
   flex-direction: row-reverse;
 }
+
+.glue-LinkEditor-widget {
+  border: solid 2px var(--jp-border-color1);
+}
+
+.glue-LinkEditor-title {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--jp-layout-color4);
+  min-height: 25px;
+  color: var(--jp-ui-inverse-font-color0);
```

Comment:
In dark mode this looks like:

![Screenshot from 2023-04-19 11-41-29](https://user-images.githubusercontent.com/21197331/233035587-534624a3-e0db-4024-a4bc-15dfbfbb79b7.png)

The following seems to be a better combination of colors for the dark theme:

```suggestion
  background-color: var(--jp-layout-color2);
  min-height: 25px;
  color: var(--jp-ui-font-color1);
```

![Screenshot from 2023-04-19 11-40-37](https://user-images.githubusercontent.com/21197331/233035336-f1b858b3-223f-4463-8c34-8b766192bc44.png)

Reply:
Good catch, thanks